src/cli/subcommands/auth_cmd.rs

```diff
@@ -2,27 +2,34 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use crate::auth::*;
-use crate::libp2p::{Multiaddr, Protocol};
 use crate::rpc_api::auth_api::AuthNewParams;
-use crate::rpc_client::auth_new;
+use crate::rpc_client::{auth_new, API_INFO};
+use chrono::Duration;
 use clap::Subcommand;
 use jsonrpc_v2::Error as JsonRpcError;
+use std::str::FromStr;
 
-use super::{handle_rpc_err, print_rpc_res_bytes, Config};
+use super::{handle_rpc_err, print_rpc_res_bytes};
 
 #[derive(Debug, Subcommand)]
 pub enum AuthCommands {
     /// Create a new Authentication token with given permission
     CreateToken {
-        /// permission to assign to the token, one of: read, write, sign, admin
+        /// Permission to assign to the token, one of: read, write, sign, admin
         #[arg(short, long)]
         perm: String,
+        /// Token is revoked after this duration
+        #[arg(long, default_value_t = humantime::Duration::from_str("2 months").expect("infallible"))]
+        expire_in: humantime::Duration,
```

Comment:
I think you know what I am going to ask you!

Reply:
I have no idea how to test it.